lib/ts/index.ts

```diff
@@ -143,6 +143,15 @@ export default class AuthHttpRequest {
             userContext: getNormalisedUserContext(userContext)
         });
     };
+
+    static getAccessToken = async (input?: { userContext?: any }): Promise<string | undefined> => {
+        // This takes care of refreshing the access token if necessary.
+        await AuthHttpRequestFetch.recipeImpl.getAccessTokenPayloadSecurely({
+            userContext: getNormalisedUserContext(input === undefined ? undefined : input.userContext)
+        });
```

Comment:
im not sure if implicitly doing this is good. If tomorrow the impl of `getAccessTokenPayloadSecurely` changes ever so slightly, then we may not rememeber to also check this function. Instead, you should do what `getAccessTokenPayloadSecurely` does.

Reply:
I'll make this call `doesSessionExist` instead. I think if we change the implementation of that so that it no longer refreshes the session, then we'd have to check all places that call that function anyway. I'd prefer not to repeat the code for this reason.